README.md

```diff
@@ -1,4 +1,57 @@
-atlantis
+Atlantis
 ========
 
-Open Source PaaS Built on Docker
+Atlantis is an Open Source PaaS for HTTP applications built on Docker and written in Go.  It makes it easy to build and deploy applications in a safe, repeatable fashion, and flexibly route requests to the appropriate containers.
+
+We're using Atlantis heavily at Ooyala for new applications; while it still has some rough edges around getting it up and running, the experience of using it for deploying applications is fairly smooth.
+
+For an introduction or more information on how to use Atlantis, see [A User's Guide to Getting Started With Atlantis](http://ooyala.github.io/atlantis/user-getting-started.html).  For more technical details on the project, continue reading here.
+
+
+This Repository
+---------------
+
+This repository is shared across all components; it contains shared datatypes and utility functions.  It is included as a submodule in the other components to make these available without code duplication.  (At the time we started Atlantis, Go package managers were generally fairly immature; it might make sense to switch to one of those, but for the moment, submodules work.)
+
+
+Architecture
+------------
+
+Atlantis consists of multiple components, each with their own repository:
+
+\[insert Cad's image here\]
```

Comment:
His architecture diagram that he drew up when the Stockholm folks were in town.  It's probably in a Google doc somewhere; we just need to export it and link it in.